Samples/ControlMonitorApp/ControlMonitorHelper/ControlMonitorHelper.h

```diff
@@ -5,12 +5,26 @@
 #pragma once
 #include "ControlMonitorManager.g.h"
 
-namespace winrt::ControlMonitorHelper::implementation
+namespace winrt::ControlMonitorHelperWinRT::implementation
 {
+    inline winrt::hstring FormatString(LPCWSTR szFormat, ...)
```

Comment:
>FormatString [](http://example.com/codeflow?start=26&length=12)

nit/tip wil has wil::str_printf that essentially does the va_args work, I don't think it directly support winrt::hstring but would support wil::unique_hstring etc

Reply:
cool, yeah I need to keep using hstring here for simplicity but will keep this API in mind